submissions/DmytryjK/HTML-CSS-Popup/sass/components/header.scss

```diff
@@ -0,0 +1,209 @@
+        }
+        &:checked + .header__popup-btn-img:after {
+            visibility: visible;
+            transition-delay: 0.3s;
+        }
+        &:checked ~ .header__popup-submenu  {
+            visibility: visible;
+            transform: translateY(21px);
+            transition: 0.3s visibility;
+            transition: 0.3s transform;
+        }
+        &:focus + .header__popup-btn-img {
+            outline: 1px solid white;
+            border-radius: 5px;
+        }
+    }
+
+    &__popup-submenu{
+        position: absolute;
+        right: 50px;
+        overflow: auto;
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+        grid-auto-rows: 1fr;
+        align-items: center;
+        justify-items: center;
+        gap: 30px;
+        padding-top: 30px;
+        height: 500px;
+        width: 105%;
+        z-index: -1;
+        list-style-type: none;
+        border: 1px solid #1C1B1E;
+        border-radius: 5px;
+        box-shadow: 1px 2px 3px grey;
+        transform: translateY(-600px);
+        visibility: hidden;
+        transition: 0.3s all;
+    }
+    &__popup-item {
+        width: 100%;
+        &:nth-child(3n+1) {
+            margin-left: 30px;
+        }
+        &:nth-child(3n+3) {
+            margin-right: 30px;
+        }
+    }
+    &__popup-link {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 5px 0;
+        font-size: 16px;
+        font-family: $ff-nav;
+        text-decoration: none;
+        border: 1px solid transparent;
+        color: $color-link;
+        &:hover {
+            padding: 5px 0;
+            filter: drop-shadow(0px 1px 2px grey);
+            border: 1px solid lightgrey;
+            border-radius: 5px;
+            transform: translateY(1px);
+            transition: 0.3s all;
+        }
+        &:focus {
+            outline: solid lightgrey;
+            border-radius: 5px; 
+        }
+        &:active {
+            outline: solid lightskyblue;
+            border-radius: 5px; 
+        }
+    }
+    &__popup-img{
+        display: block;
+        width: 50px;
+        height: 40px;
+        object-fit: cover;
+    }
+
+
+    &__popup-more {
+        align-self: end;
+        grid-column-start: 1;
+        grid-column-end: 4;
+        height: 40px;
+        width: 100%;
+        background-color: lightgrey;
+    }
+
+    &__popup-input-more {
+        appearance: none;
+        display: block;
+        &:checked ~ .header__more-content  {
+            display: grid;
+            grid-template-columns: repeat(3, 1fr);
+            grid-template-rows: repeat(2, 1fr);
+            grid-auto-rows: 1fr;
+            align-items: center;
+            justify-items: center;
+            gap: 30px;
+            padding: 40px 0;
+        }   
+        &:focus + .header__popup-btn-text  {
+            background-color: lightblue;
```

Comment:
If you use CSS custom properties for colors use this approach throughout the project.